GB1 fails with ClassNotFoundException GradebookEntityProducer

This happens because the package on class was incorrect and the spring application context was not updated.
```gradebook.ui.helpers.entity.GradebookEntityProvider) and 6 capabilities
19-Aug-2019 11:11:19.873 ERROR [Catalina-utility-4] org.springframework.web.context.ContextLoader.initWebApplicationContext Context initialization failed
org.springframework.beans.factory.CannotLoadBeanClassException: Cannot find class [org.sakaiproject.gradebook.framework.GradebookEntityProducer] for bean with name 'org.sakaiproject.gradebook.framework.GradebookEntityProducer' defined in ServletContext resource [/WEB-INF/applicationContext.xml]; nested exception is java.lang.ClassNotFoundException: org.sakaiproject.gradebook.framework.GradebookEntityProducer
    at org.springframework.beans.factory.support.AbstractBeanFactory.resolveBeanClass(AbstractBeanFactory.java:1391)
    at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.determineTargetType(AbstractAutowireCapableBeanFactory.java:639)
    at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.predictBeanType(AbstractAutowireCapableBeanFactory.java:607)
    at org.springframework.beans.factory.support.AbstractBeanFactory.isFactoryBean(AbstractBeanFactory.java:1490)
    at org.springframework.beans.factory.support.AbstractBeanFactory.isFactoryBean(AbstractBeanFactory.java:1013)
    at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:741)
    at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:867)
    at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:543)
    at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:443)
    at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:325)
    at org.sakaiproject.util.SakaiContextLoaderListener.contextInitialized(SakaiContextLoaderListener.java:56)
    at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4682)
    at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5150)
    at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
    at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:713)
    at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:690)
    at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:695)
    at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:978)
    at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1850)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
    at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
    at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassNotFoundException: org.sakaiproject.gradebook.framework.GradebookEntityProducer
    at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1363)
    at org.apache.catalina.loader.WebappClassLoaderBase.loadClass(WebappClassLoaderBase.java:1186)
    at org.springframework.util.ClassUtils.forName(ClassUtils.java:251)
    at org.springframework.beans.factory.support.AbstractBeanDefinition.resolveBeanClass(AbstractBeanDefinition.java:401)
    at org.springframework.beans.factory.support.AbstractBeanFactory.doResolveBeanClass(AbstractBeanFactory.java:1438)
    at org.springframework.beans.factory.support.AbstractBeanFactory.resolveBeanClass(AbstractBeanFactory.java:1383)
    ... 26 more```

--- a/app/sakai-tool/src/webapp/WEB-INF/applicationContext.xml
+++ b/app/sakai-tool/src/webapp/WEB-INF/applicationContext.xml
@@ -65,7 +65,7 @@
 
     <bean
         id="org.sakaiproject.gradebook.framework.GradebookEntityProducer"
-        class="org.sakaiproject.gradebook.framework.GradebookEntityProducer"
+        class="org.sakaiproject.tool.gradebook.framework.GradebookEntityProducer"
         init-method="init">
         <property name="entityManager" ref="org.sakaiproject.entity.api.EntityManager" />
         <property name="gradebookService" ref="org.sakaiproject.service.gradebook.GradebookService" />
@@ -74,7 +74,7 @@
 
     <bean
         id="org.sakaiproject.gradebook.framework.GradebookContextObserver"
-        class="org.sakaiproject.gradebook.framework.GradebookContextObserver"
+        class="org.sakaiproject.tool.gradebook.framework.GradebookContextObserver"
         parent="org.sakaiproject.gradebook.framework.GradebookEntityProducer">
     </bean>
     <!-- EB end -->

